### PR TITLE
[WI-46754] Add ext-uv stubs

### PR DESCRIPTION
**Pull request by WyriHaximus**
Resolves: https://youtrack.jetbrains.com/issue/WI-46754

### CODE REVIEW

File: uv/uv_functions.php [hunk 1]
```diff
@@ -0,0 +1,1336 @@
...
+ * @return void
+ */
+function uv_check_start(resource $handle, callable $callback): void {}
+
+/**
+ * Stop check callback.
+ *
+ * @param resource $handle uv resource handle (check).
+ *
+ * @return void`
```
Comment (MaXal) [inline uv/uv_functions.php]: Please change to `@return void` (remove the closing `)

Comment (WyriHaximus) [reply uv/uv_functions.php]: For a moment I was pretty impressed you managed to spot something I thought was a bit of dust. Then I realised the tests failed :joy:

Comment (WyriHaximus) [reply uv/uv_functions.php]: Ok fixed it, messed up rebase, fixed that as well

**Comment by MaXal** [comment]
For future, UV extension requires PHP 7.0 but you made stubs PHP 7.1 only compatible. I've fixed that otherwise, people will get error code navigating to the stubs.

Please also note that `resource` is not a primitive type (removed this as well).

**Comment by WyriHaximus** [comment]
Will keep that in mind fix next time 👍 !

**Pull request merged**